mmv1/third_party/terraform/website/docs/r/google_project.html.markdown

```diff
@@ -98,6 +98,11 @@ The following arguments are supported:
     `false`. Note that when `false`, Terraform enables `compute.googleapis.com` on the project to interact
     with the GCE API and currently leaves it enabled.
 
+* `deletion_policy` -  (Optional) The deletion policy for the Project. Setting PREVENT will protect the project
+   against any destroy actions caused by a terraform apply or terraform destroy. Setting ABANDON allows the resource 
+   to be abandoned rather than deleted, i.e., the Terraform resource can be deleted without deleting the Project via 
+   the Google API. Possible values are: "PREVENT", "ABANDON", "NONE". Default value is `PREVENT`.
+
```

Comment:
`skip_delete` should be removed from line 81.

Reply:
Removed. Thanks.